internal/ui/ui_mobile.go

```diff
@@ -428,6 +429,26 @@ func (u *userInterfaceImpl) Window() Window {
 	return &nullWindow{}
 }
 
+type Monitor struct{}
+
+func (m *Monitor) Bounds() image.Rectangle {
+	// TODO: This should return the available viewport dimensions.
+	return image.Rectangle{}
+}
+
+func (m *Monitor) Name() string {
+	return ""
+}
+
+func (u *userInterfaceImpl) AppendMonitors(mons []*Monitor) []*Monitor {
+	return nil
```

Comment:
```go
return append(mons, &Monitor{})
```

Reply:
To all of these, we should make `&Monitor{}` a global constant, so that `ebiten.Monitor()` can be compared to the dummy monitor acquired from `AppendMonitors` successfully.

Reply:
e.g.,

```golang
const dummyMonitor = &Monitor{}

func (u *userInterfaceImpl) AppendMonitors(mons []*Monitor) []*Monitor {
  append(mons, dummyMonitor)
}
```

Reply:
You are right. Then, let's define `var theMonitor Monitor` and use it.

Reply:
I guess it has to be var, not const for pointers*